package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "hexo": {
-    "version": "3.3.1"
+    "version": "3.8.0"
```

Comment:
This was auto-bumped when I installed packages.  I assumed this was okay.  It seemed to work while developing locally.